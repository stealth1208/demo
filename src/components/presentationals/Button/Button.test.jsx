import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '../index';

it('renders correctly', () => {
    const tree = renderer
      .create(<Button type="button" onClick={() => console.log('Click')}>On Click</Button>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });