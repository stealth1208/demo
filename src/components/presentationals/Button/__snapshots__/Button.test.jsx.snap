// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="primary-button undefined"
  onClick={[Function]}
  type="button"
>
  On Click
</button>
`;
